test/tests/pbs_smoketest.py

```diff
@@ -79,15 +79,38 @@ def test_submit_job_array(self):
 
     def test_advance_reservation(self):
         """
-        Test to submit an advanced reservation
+        Test to submit an advanced reservation and submit jobs to that
+        reservation. Check if the reservation gets confimed and the jobs
+        inside the reservation starts running when the reservation runs.
         """
-        r = Reservation()
-        a = {'Resource_List.select': '1:ncpus=1'}
+        a = {'resources_available.ncpus': 4}
+        self.server.manager(MGR_CMD_SET, NODE, a, id=self.mom.shortname)
```

Comment:
If you do this, I think you need to add the skip on cpuset decorator to this test.  You can't modify the resources on a cpuset machine to differ from what the mom reported.